// pdf-parse/dist/cjs/geometry/Geometry.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Table = exports.Shape = exports.Rectangle = exports.Point = exports.LineStore = exports.LineDirection = exports.Line = void 0;
var Line_js_1 = require("./Line.js");
Object.defineProperty(exports, "Line", { enumerable: true, get: function () { return Line_js_1.Line; } });
Object.defineProperty(exports, "LineDirection", { enumerable: true, get: function () { return Line_js_1.LineDirection; } });
var LineStore_js_1 = require("./LineStore.js");
Object.defineProperty(exports, "LineStore", { enumerable: true, get: function () { return LineStore_js_1.LineStore; } });
var Point_js_1 = require("./Point.js");
Object.defineProperty(exports, "Point", { enumerable: true, get: function () { return Point_js_1.Point; } });
var Rectangle_js_1 = require("./Rectangle.js");
Object.defineProperty(exports, "Rectangle", { enumerable: true, get: function () { return Rectangle_js_1.Rectangle; } });
var Shape_js_1 = require("./Shape.js");
Object.defineProperty(exports, "Shape", { enumerable: true, get: function () { return Shape_js_1.Shape; } });
var Table_js_1 = require("./Table.js");
Object.defineProperty(exports, "Table", { enumerable: true, get: function () { return Table_js_1.Table; } });
